2-Webpack/main.js

```diff
@@ -0,0 +1,180 @@
+const { v4: uuidv4 } = require("uuid");
+import { LocalStorage } from "./utils/localstorage";
```

Comment:
do not mix es6 and commonjs imports, for client/browser project we'd usually go with es6 imports

Reply:
Alright, fixed it.